cache/src/main/java/com/playone/mobile/cache/PlayoneCacheImpl.kt

```diff
@@ -0,0 +1,127 @@
+package com.playone.mobile.cache
+
+import android.database.sqlite.SQLiteDatabase
+import com.playone.mobile.cache.db.Db
+import com.playone.mobile.cache.db.DbOpenHelper
+import com.playone.mobile.cache.db.constants.BufferooConstants
+import com.playone.mobile.cache.db.mapper.BufferooMapper
+import com.playone.mobile.cache.db.mapper.PlayoneMapper
+import com.playone.mobile.cache.mapper.BufferooEntityMapper
+import com.playone.mobile.cache.model.CachedBufferoo
+import com.playone.mobile.data.model.BufferooEntity
+import com.playone.mobile.data.repository.BufferooCache
+import io.reactivex.Completable
+import io.reactivex.Single
+
+/**
+ * Cached implementation for retrieving and saving Bufferoo instances. This class implements the
+ * [BufferooCache] from the Data layer as it is that layers responsibility for defining the
+ * operations in which data store implementation layers can carry out.
+ */
+class PlayoneCacheImpl constructor(dbOpenHelper: DbOpenHelper,
+                                   private val entityMapper: BufferooEntityMapper,
+                                   private val mapper: PlayoneMapper,
+                                   private val preferencesHelper: PreferencesHelper):
+        BufferooCache {
```

Comment:
It should be `PlayoneCache`, right?!

Reply:
Oh.. as I mentioned, just ignore cache layer for now..